demos/README.md

```diff
@@ -88,9 +94,15 @@ To be able to build demos you need to source Inference Engine and OpenCV environ
 Please run the following command before the demos build (assuming that the binary package was installed to `<INSTALL_DIR>`):
 
 ```sh
-source <INSTALL_DIR>/deployment_tools/bin/setupvars.sh
+<INSTALL_DIR>/extras/scripts/download_opencv.sh
+source <INSTALL_DIR>/setupvars.sh
 ```
 
+> **NOTE:** If you plan to use Python\* demos only, you can install the OpenVINO Python\* package.
+> ```sh
+> pip install openvino
+> ```
+
 You can also build demos manually using Inference Engine built from the [openvino](https://github.com/openvinotoolkit/openvino) repo. In this case please set `InferenceEngine_DIR` environment variable to a folder containing `InferenceEngineConfig.cmake` and `ngraph_DIR` to a folder containing `ngraphConfig.cmake` in a build folder. Please also set the `OpenCV_DIR` to point to the OpenCV package to use. The same OpenCV version should be used both for Inference Engine and demos build. Alternatively these values can be provided via command line while running `cmake`. See [CMake's search procedure](https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure).
```

Comment:
There still references to Inference Engine and `InferenceEngine_DIR`

Reply:
updated references to Inference Engine, except `InferenceEngine_DIR` is an environment variable to `InferenceEngineConfig.cmake` file directory

Reply:
@jkamelin I think InferenceEngineConfig.cmake still maintained for needs of applications built on old API. Suggest to advice customers to stick to new API instead.
@ilya-lavrenov seems the page https://github.com/openvinotoolkit/openvino/wiki/BuildingCode outdated, could you please advice on how we need to describe build OpenVINO ?

Reply:
according to @ilya-lavrenov now it should be OpenVINO_DIR

Reply:
right, instead of ngraph and IE configs we need to use `OpenVINOConfig.cmake`, in cmake cmd line we need to point `OpenVINO_DIR` location.
As a target to link application with - use `openvino::runtime` instead of ngraph and IE ones

Reply:
Ok, looks like we need to update our demos first, because this variables are used to build them.